platform/mellanox/sdk-src/sx-kernel/0001-Disable-MLXFW-module.patch

```diff
@@ -0,0 +1,25 @@
+From 0188d76b14cd9d681849f0e7966fe188ad831d77 Mon Sep 17 00:00:00 2001
```

Comment:
Patch to disable MLXFW in SDK which conflicts with kernel module. This is resolved in SDK upstream, we are waiting on a release from them to upgrade to before regression, until then this patch is included to allow building.